artio-system-tests/src/test/java/uk/co/real_logic/artio/system_tests/PersistentSequenceNumberGatewayToGatewaySystemTest.java

```diff
@@ -1134,6 +1147,16 @@ private void exchangeMessagesAroundARestart(
             assertSequenceFromInitToAcceptAt(expectedInitToAccSeqNum, expectedAccToInitSeqNum);
         }
 
+        // this means a resend request will be sent by the acceptor
+        if (initialSentSequenceNumber > expectedAccToInitSeqNum)
```

Comment:
Aren't you comparing sequence numbers across two different directions?

Reply:
yeh - am fixing the logic - thx